src/normalize.js

```diff
@@ -111,11 +114,16 @@ const normalizeReferenceField = (value, referenceTo, locale, entries,  createNod
     return reference;
 }
 
+const getSchemaValue = (obj, key) => {
+    if (obj === null) return null;
+    if (typeof obj !== "object") return null;
+    return obj.hasOwnProperty(key.uid) ? obj[key.uid] : null;
+};
 
 const builtEntry = (schema, entry, locale, entries, createNodeId) => {
 	let entryObj = {};
     schema.forEach(field => {
-        let value = (typeof entry[field.uid] != 'undefined') ? entry[field.uid] : null;
+        const value = getSchemaValue(entry, field);
```

Comment:
and here I added some extra levels of checks to ensure we aren't dealing with a null